src/components/PhotoAlbum.css

```
@@ -0,0 +1,24 @@
+ul {
```

```suggestion
.album {
    ul {
```
Recommend sticking to the modular css classes like above moving forward :)